app.js

```diff
@@ -43,7 +43,7 @@ app.post('/api/v1/projects', async (request, response) => {
   }
   try {
     const id = await database('projects').insert(project, 'id');
-    response.status(201).json({ id });
+    response.status(201).json({ id: id[0] });
```

Comment:
Thank you!   You may need to change the corresponding test from `response.body.id[0]` to just `response.body.id`

Reply:
@hfaerber Good call! I pushed 2 new commits, the first changes the response to what you requested and the other fixes the test that this change broke.